edgepi-digital-out.js

```diff
@@ -7,13 +7,14 @@ module.exports = function(RED) {
     function DigitalOutNode(config) {
         RED.nodes.createNode(this, config);
         const node = this;
+        // Get node properties
         node.channelType = config.channelType;
         node.powerOn = config.powerOn;
 
         function inputlistener(msg, send, done) {
             if (node.child != null) {
                 // send input to child process using stdin (Py script)
-                node.child.stdin.write(command+"\n", () => {
+                node.child.stdin.write(this.channelType+","+this.powerOn+"\n", () => {
```

Comment:
I think we could use a template string here like:

```js
node.child.stdin.write(`${this.channelType}, ${this.powerOn}\n`, () => {
  if (done) { done(); }
});
```

Reply:
88092acf17fe0a1aebb1a46fbb9f0d56583eb580 resolves with your exact code.